samples/nrf5340/multicore/README.rst

```diff
@@ -0,0 +1,65 @@
+.. _nrf5340_multicore:
+
+nRF5340: Multi core application
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can use this sample to see how to build an application with the network core build as child image from soruce.
```

Comment:
Just a thought.  Shouldn't we say that we can use this as a sample for running application on any Nordic SoC with 2 cores. It will be ready for Haltium SoCs.
  "... can use this sample to see how to build application on two cores. In this example we use..."